include/seqan3/alignment/pairwise/policy/affine_gap_banded_policy.hpp

```diff
@@ -72,25 +72,53 @@ class affine_gap_banded_policy : public affine_gap_policy<affine_gap_banded_poli
     {
         using std::get;
         // Unpack the cached variables.
-        auto & [current_entry, next_entry] = get<0>(current_cell);
-        auto & [main_score, hz_score] = current_entry;
-        auto const & prev_hz_score = get<1>(next_entry);
+        auto & [score_entry, coordinate, trace_value] = current_cell;
+        auto & [current_entry, next_entry] = score_entry;
+        auto & [main_score, hz_score, hz_trace] = current_entry;
+        auto const & [_ignore, prev_hz_score, prev_hz_trace]= next_entry;
         auto & [prev_cell, gap_open, gap_extend, opt] = cache;
-        auto & [tmp, vt_score] = prev_cell;
+        auto & [tmp, vt_score, vt_trace] = prev_cell;
+
+        std::ignore = _ignore;
 
         //TODO For the local alignment we might be able to use GCC overlow builtin arithmetics, which
         // allows us to check if overflow/underflow would happen. Not sure, if this helps with the performance though.
         // (see https://gcc.gnu.org/onlinedocs/gcc/Integer-Overflow-Builtins.html)
         main_score += score;
-        main_score = std::max(main_score, vt_score);
-        main_score = std::max(main_score, prev_hz_score);
+        // Compute where the max comes from.
+        if constexpr (decays_to_ignore_v<decltype(trace_value)>) // Don't compute a traceback
+        {
+            main_score = (main_score < vt_score) ? vt_score : main_score;
+            main_score = (main_score < prev_hz_score) ? prev_hz_score : main_score;
```

Comment:
use `std::max(main_score, std::max(vt_score, prev_hz_score))`

Reply:
well in SeqAn2 it made a performance difference, also I am not sure why.
Second it is likely that I remove this when dealing with the performance thing, as I hope that in case trace_value is std::ignore, assigning to it will be optimised away